### PR TITLE
OC-9605: Query notifications to include item label/description

### PR DESCRIPTION
**Pull request by rooney**
https://jira.openclinica.com/browse/OC-9605

### CODE REVIEW

File: src/main/java/core/org/akaza/openclinica/domain/datamap/Item.java [hunk 1]
```diff
@@ -168,6 +169,14 @@ public void setDescription(String description) {
 		this.description = description;
 	}
 
+    @Column(name = "brief_description")
+    public String getBriefDescription() {
```
Comment (svadla-oc) [inline src/main/java/core/org/akaza/openclinica/domain/datamap/Item.java]: why are you adding a new column?

Comment (svadla-oc) [reply src/main/java/core/org/akaza/openclinica/domain/datamap/Item.java]: If we need to add a new column, we need to create liquibase script. 
@jkeremian can you review this please?

Comment (rooney) [reply src/main/java/core/org/akaza/openclinica/domain/datamap/Item.java]: The column is already in the DB, just not exposed in the bean, so no need to liquibase
CMIIW

**Comment by rooney** [comment]
https://jira.openclinica.com/browse/OC-9605 (AC1 & 2)

**Comment by svadla-oc** [comment]
jenkins retest this please

**Comment by svadla-oc** [comment]
👍

**Pull request merged**